DESCRIPTION

```diff
@@ -29,10 +29,10 @@ URL: https://github.com/insightsengineering/rtables,
     https://insightsengineering.github.io/rtables/
 BugReports: https://github.com/insightsengineering/rtables/issues
 Depends:
-    formatters (>= 0.5.6),
+    R (>= 2.10),
```

Comment:
This is a general discussion but it seems a very low number to me. We might want to move towards dropping magrittr in favor of native |>

Reply:
lets do it in the future